src/app/components/JumpTo/index.tsx

```diff
@@ -23,38 +23,36 @@ const JumpTo = ({ jumpToHeadings, eventTrackingData }: JumpToProps) => {
     componentName: 'jumpto',
   });
 
-  const headingId = 'jump-to-heading';
+  const heading = 'jump-to-heading';
 
   // we use the Text component with the as prop set to strong (for now) because the screenreader UX states the heading should not be announced
   // using InlineLink instead of anchor to bring benefits to styling, but can revert to anchor if needed
   return (
-    <section
+    <nav
```

Comment:
Don't think this should be `nav`

Reply:
Not sure what the discussion and change around this was whilst I was OOO - should I revert to `section` or something else?

Reply:
@greenc05 mentioned to remove the `nav`, which I take to mean remove entirely? https://github.com/bbc/simorgh/pull/12123#discussion_r1827944699

Reply:
The comment you referenced there @amoore108 is to remove that specific `nav` tag on line 40 and the conversation @greenc05 and I had was to change the `section` on line 31 to `nav`

Reply:
Ah cool, thanks 👍